.travis.yml

```diff
@@ -5,11 +5,23 @@ python:
   - 3.3
   - 3.4
   - 3.5
-  - pypy
-  - pypy3
-install:
-  - pip install tox-travis
+  - 3.6
+  - pypy-5.4.1
```

Comment:
The default version of PyPy 3 on travis only implements Python 3.2, and pip and other things no longer work with it. 

There is a 3.3-alpha version you can specifically request, but that's a dead end: there will be no more development of it in favor of a PyPy3.5 (which is now in beta, but is not available on Travis without special effort).

Between regular PyPy2 and the various 3.x tests, I feel like we're pretty well covered, with the exception of possible beta bugs, which I'd rather not spend any time debugging.